nc/source/graphics/TextureManager.cpp

```diff
@@ -29,94 +34,50 @@ namespace nc::graphics
 
     void TextureManager::LoadTextures(const std::vector<std::string>& paths)
     {
-        auto imagesCount = paths.size();
-
-        std::vector<vk::DescriptorImageInfo> imageInfos = {};
-        std::vector<ImmutableImage> images = {};
-        std::unordered_map<std::string, uint32_t> accessors = {};
-        std::vector<std::string> pathsToLoad;
-
-        auto texturesExist = ResourceManager::HasTextures();
-        if (texturesExist)
-        {
-            pathsToLoad = ResourceManager::GetTexturePaths();
-            pathsToLoad.insert(paths.end(), paths.begin(), paths.end());
-        }
-        else
-        {
-            pathsToLoad = paths;
-        }
-
         auto sampler = m_graphics->GetBasePtr()->CreateTextureSampler();
```

Comment:
I'm not sure what the common practice is with these? Should they be created and thrown away as needed or created once and reused?